packages/components/src/shared/artplayer.ts

```diff
@@ -0,0 +1,72 @@
+import type artplayer from "artplayer/dist/artplayer.js";
+
+export type ArtPlayer = artplayer;
+export type ArtPlayerOptions = typeof artplayer.option;
+
+type ArtPlayerCustomType = typeof artplayer.option.customType;
+
+export interface CustomArtPlayerOptions
+  extends Omit<ArtPlayerOptions, "container" | "url" | "customType"> {
+  container?: HTMLElement | string;
+  url?: string;
```

Comment:
JSDocs missing, it's hard for me to review in github, if `container` is meant to be fixed at component side, this should not be exposed to users.